Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageBubbleView.swift

```diff
@@ -5,22 +5,69 @@
 import StreamChat
 import UIKit
 
-open class ChatMessageBubbleView: _View, AppearanceProvider {
-    open var roundedCorners: CACornerMask = .all {
+/// A view that displays a bubble around a message.
+public typealias ChatMessageBubbleView = _ChatMessageBubbleView<NoExtraData>
+
+/// A view that displays a bubble around a message.
+open class _ChatMessageBubbleView<ExtraData: ExtraDataTypes>: _View, AppearanceProvider, SwiftUIRepresentable {
```

Comment:
I kinda like the `bubble` terminology since it's commonly used to refer to the thing drawn around the message (no matter what shape it has) 😄 

The name I was considering is `ChatMessageBackgroundView` but it has no soul 🤷

Reply:
Yes, I totally agree with you, `Bubble` is very easily recognizable. And for that reason, it is the right choice. Bubble is fine for me then 👍